cmd/abigen/main.go

```diff
@@ -156,5 +166,10 @@ func contractsFromStdin() (map[string]*compiler.Contract, error) {
 		return nil, err
 	}
 
-	return compiler.ParseCombinedJSON(bytes, "", "", "", "")
+	code, err := compiler.ParseCombinedJSON(bytes, "", "", "", "")
+	if err != nil {
```

Comment:
No need to trap and print this error here, the method returns an error already, so just bubble it up to the caller instead (like it did previously)